### ISSUE TITLE
ubuntu question: comming soon :)

### ISSUE THREAD

**Issue by joeblew99** [openstf/setup-examples#2]
i am using coreos and also ubuntu snappy core.
Ubuntu snappy core is the new way of doing containers and managing apps on ubuntu.
CoreOS is awesome, but ubuntu snappy core runs on embedded and servers withthe transactional updates that coreos gives.
So, ubuntu snappy core provides more long term roadmap possibilities, for when you know you need to deploy to servers,, clients and embedded.

so, my question is will you be targeting ubuntu snappy ?

**Comment by sorccu** [comment]
Should be fairly easy as long as there's systemd and docker support.
Personally I'm using Arch Linux right now, it works great on many ARM
devices too. Feel free to contribute instructions for snappy!

**Comment by joeblew99** [comment]
Well in your roadmap you guys said you were targeting Ubuntu. So I am asking if you plan to target Ubuntu core snappy ?

**Comment by vbanthia-zz** [comment]
Yeah, providing Ubuntu sample is in roadmap but we are not sure when we can do this.
As Simo has said, the only requirement for deploying STF is docker and some tool to start unit such as Systemd. So it will be really easy to deploy stf on ubuntu core.

We would like to see stf working on snappy too. In case if you have any problem, we would be happy to help you in troubleshooting :)

**Comment by joeblew99** [comment]
Yes I am only targeting Ubuntu core snappy and will be using it across all of our projects. We are a company targeting IOT style situations and often have 1000 of devices out in the field. These are servers, workstations and embedded devices. The embedded devices use WiFi, BT. 
So for us its a clear winner.
The only thing I see missing is how to provision is upgrades and application updates when these devices only have http access. Pxe and SSH is not possible for us.

**Comment by joeblew99** [comment]
At the moment we have a hacked together mixture of technologies that vets the job done, but moving to a platform like Ubuntu core snappy removes allot of the cruft

**Comment by joeblew99** [comment]
Because so many of our devices are mobiles, we need to be able test them. With Ubuntu core snappy running the testing it u ifirs our architecturry. Hope this all makes sense.

**Comment by FranklinYu** [comment]
Is there a simpler setup guide for Ubuntu if I don't need a cluster? I only have one server that host both the database (RethinkDB) and the web application. I think many things can be simplified, for example database should not need password since it is only exposed to local network. Reliability is not a strong requirement, but we want something more than `stf local` which does not support domain name (or does it?).
